Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Faq.rst

```diff
@@ -65,19 +65,7 @@ How to convert to the new language XLIFF file format
 
 If you have :ref:`downloaded an XLIFF file <migrate-from-pootle>` from the
```

Comment:
Pootle Server has been shut down completely. I guess this is obsolete

Reply:
That should be a separate PR then, as this change is about the Core change